packages/teams-js/src/public/pages.ts

```diff
@@ -1028,10 +1029,12 @@ export namespace pages {
     }
 
     /**
-     * Navigate within the currently running application with page ID, and sub-page ID (for navigating to
-     * specific content within the page).
-     * @param params - Parameters for the navigation
-     * @returns a promise that will resolve if the navigation was successful
+     * Navigate within the currently running app
+     *
```

Comment:
Given doc rendering, you might consider not having the blank line or having the second line be a `@remark`.